radicle-link-test/src/integration/git_helpers/remote.rs

```diff
@@ -130,13 +131,18 @@ fn credential_helper() -> String {
 }
 
 fn setup_path() -> anyhow::Result<PathBuf> {
-    let helper_path = env!("CARGO_BIN_EXE_git-remote-rad");
-    let helper_path = Path::new(helper_path).parent().unwrap();
+    // The git-remote-rad binary is located under radicle-link/target/debug
```

Comment:
This seems to work only _sometimes_ XD

I think we should just make the test crate build the remote helper binary, so cargo can find it just as before.

Reply:
Boooooo :(

I tried that, and I ran into some issues. I didn't keep track of what they were so I'll give it another shot.

Reply:
Do you know if there was anything special about how you triggered this?

And also I don't think we can use it as before because that's specifically for integration tests:

> CARGO_BIN_EXE_<name> — The absolute path to a binary target's executable. This is only set when building an integration test or benchmark. This may be used with the env macro to find the executable to run for testing purposes. The <name> is the name of the binary target, exactly as-is. For example, CARGO_BIN_EXE_my-program for a binary named my-program. Binaries are automatically built when the test is built, unless the binary has required features that are not enabled.

Reply:
> integration test or benchmark

Ah dang

Reply:
Oh I see why it didn't work: one has to invoke `cargo build` before `cargo test`, because the latter won't make sure the binary was built before.

Reply:
I thought "Cool, I'll just add the `[[bin]]` to the toml and `cargo test` should build it too", but nope :(

Reply:
Ok, so hear me out... What about:

```rust
    if !helper_path.join("git-remote-rad").exists() {
        std::process::Command::new("cargo")
            .args(&["build", "--bin", "git-remote-rad"])
            .output()?;
    }
```

Reply:
Turns out that that doesn't work either :grimacing:

Reply:
:roller_coaster: Oh wait it does! If we add it to the `Cargo.toml` _and_ do this it will build the binary if it's missing